Source/WebCore/html/HTMLMediaElement.h

```diff
@@ -1288,8 +1288,8 @@ class HTMLMediaElement
     bool m_tracksAreReady : 1;
     bool m_haveVisibleTextTrack : 1;
     bool m_processingPreferenceChange : 1;
-    bool m_shouldAudioPlaybackRequireUserGesture : 1;
-    bool m_shouldVideoPlaybackRequireUserGesture : 1;
+    bool m_shouldAudioPlaybackRequireUserGesture : 1 { true };
```

Comment:
Initializing bit fields with initializer lists?  Has technology really come this far?  Back in my day this was impossible.

Reply:
👍